imports/plugins/core/router/client/app.js

```diff
@@ -1,116 +1,15 @@
-import React, { Component } from "react";
-import PropTypes from "prop-types";
-import classnames from "classnames";
-import { Switch } from "react-router-dom";
-import { composeWithTracker } from "@reactioncommerce/reaction-components";
-import { Reaction, Router } from "/client/api";
-import ToolbarContainer from "/imports/plugins/core/dashboard/client/containers/toolbarContainer";
-import Toolbar from "/imports/plugins/core/dashboard/client/components/toolbar";
-import { ActionViewContainer, PackageListContainer } from "/imports/plugins/core/dashboard/client/containers";
-import { ActionView, ShortcutBar } from "/imports/plugins/core/dashboard/client/components";
-
-const ConnectedToolbarComponent = ToolbarContainer(Toolbar);
-const ConnectedAdminViewComponent = ActionViewContainer(ActionView);
-const ConnectedShortcutBarContainer = PackageListContainer(ShortcutBar);
-
-const styles = {
-  customerApp: {
-    width: "100%"
-  },
-  adminApp: {
-    width: "100%",
-    height: "100vh",
-    display: "flex",
-    overflow: "hidden"
-  },
-  adminContentContainer: {
-    flex: "1 1 auto",
-    height: "100vh"
-  },
-  adminContainer: {
-    display: "flex",
-    flex: "1 1 auto"
-  },
-  scrollableContainer: {
-    overflow: "auto"
-  }
+import React from "react";
+import { Session } from "meteor/session";
+import { getComponent } from "@reactioncommerce/reaction-components";
+
+const App = (props) => {
+  // Make sure a default value is set.
+  const defaultContainer = Session.get("DEFAULT_APP_CONTAINER") || "CoreReactionApp";
+  const AppContainer = getComponent(defaultContainer);
+
+  return (
+    <AppContainer {...props} />
```

Comment:
This file doesn't need to exist anymore. In `browserRouter.js` file in the same folder, you could just do the following.

```
import { Components } from "@reactioncommerce/reaction-components";

<Components.App>
```